apps/supernova/src/components/alerts/AlertsTab.tsx

```diff
@@ -23,12 +23,34 @@ const AlertsTab = () => {
 
   // Fetch alerts data
   const { data, isLoading, error } = useBoundQuery<AlertsData>("alerts")
+
   if (error) {
+    // Extra CORS warning based on instanceof
+    if (error instanceof CorsNetworkError) {
+      addMessage({
+        variant: "warning",
+        text: (
+          <p>
```

Comment:
It might make sense to extract this into a separate component. Just a suggestion - it's not a blocker!